app/assets/javascripts/tagging.js

```diff
@@ -80,3 +85,25 @@ function initTagForm(deletion_path, selector) {
   return el;
 
 }
+
+function geocodeStringAndPan(string, onComplete) {
+  var url = "https://maps.googleapis.com/maps/api/geocode/json?address=" + string.split(" ").join("+");
+  var Blurred = $.ajax({
+      async: false,
+      url: url,
+      complete: onComplete(data.responseJSON.results[0].geometry.location),
```

Comment:
Awesome. Have you confirmed that this works locally? If so, I think this is ready to go!

Reply:
I tried it as a sample in console, but couldn't test the tags as I told you I couldn't figure how to test it. But onComplete runs after the ajax call thats for sure.